sass/_header.scss

```diff
@@ -0,0 +1,141 @@
+// COLOR VARIABLES
+$color-primary: #28352f;
+$color-grey-dark-2: #28352f;
+
+// FONT VARIABLES
+$font-primary: 'Nunito', sans-serif;
+$font-display: 'Josefin Sans', sans-serif;
+
+// overall display setting
+*,
+*::before,
+*::after {
+  margin: 0;
+  padding: 0;
+  box-sizing: inherit;
+}
+
+html {
+  box-sizing: border-box;
+  font-size: 62.5%; // 10px/16px = 62.5% -> 1rem = 10px
```

Comment:
[OPTIONAL] Kindly remove the commented code as it makes your codebase not professional.